keras_maskrcnn/utils/coco_eval.py

```diff
@@ -0,0 +1,104 @@
+"""
+Copyright 2017-2018 Fizyr (https://fizyr.com)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from __future__ import print_function
+
+from pycocotools.coco import COCO
+from pycocotools.cocoeval import COCOeval
+from pycocotools import mask as mask_utils
+
+import numpy as np
+import json
+import os
+import cv2
+
+def evaluate_coco(generator, model, threshold=0.05):
+    # start collecting results
+    results = []
+    image_ids = []
+    for index in range(10): #generator.size()):
+        image = generator.load_image(index)
+        image_shape = image.shape
+        image = generator.preprocess_image(image)
+        image, scale = generator.resize_image(image)
+
+        # run network
+        _, _, _, boxes, classification, masks = model.predict_on_batch(np.expand_dims(image, axis=0))
```

Comment:
Please rename classification to nms_classification. I'm doing the same in retinanet.